language/if_spec.rb

```diff
@@ -1,6 +1,25 @@
 require File.expand_path('../../spec_helper', __FILE__)
 
 describe "The if expression" do
+  ruby_version_is '2.4' do
+    describe "multiple assignments in conditional expression" do
```

Comment:
Could you prepend `accepts` in the description? So it would read better when run with `mspec -fp`.